src/components/views/rooms/RoomHeader.tsx

```diff
@@ -40,10 +41,14 @@ export default function RoomHeader({ room, oobData }: { room?: Room; oobData?: I
         >
             {room && <DecoratedRoomAvatar room={room} oobData={oobData} avatarSize={40} displayBadge={false} />}
             <div className="mx_RoomHeader_info">
-                <div dir="auto" title={roomName} role="heading" aria-level={1} className="mx_RoomHeader_name">
+                <Body as="div" size="lg" weight="semibold" dir="auto" title={roomName} role="heading" aria-level={1}>
```

Comment:
Body is a really poor name IMO, could we import it as `BodyText` maybe?

Reply:
That probably won't be fixed in this pull request, but rather as part of github.com/vector-im/compound
But can definitely hear the argument that we want to change that name

Reply:
https://github.com/vector-im/compound/issues/194